contracts/implementations/RMRKNestableImpl.sol

```diff
@@ -54,7 +53,7 @@ contract RMRKNestableImpl is IRMRKInitData, RMRKAbstractNestableImpl {
     function mint(
         address to,
         uint256 numToMint
-    ) public payable virtual notLocked saleIsOpen {
```

Comment:
Why is this removed? To be less opinionated or just to preserve contract size?

Reply:
`saleIsOpen` was checking that  `totaSupply <= maxSupply`. I noticed that the modifier was never used and when tracking it it happens that on minting we check that the current supply + the total being minted doesn't go over the max supply.  So the check on `saleIsOpen` is non sense, it will never be reached.